api/issues.types.ts

```diff
@@ -4,13 +4,19 @@ export enum IssueLevel {
   error = "error",
 }
 
+export enum IssueStatus {
+  unresolved = "unresolved",
+  resolved = "resolved",
+}
+
 export type Issue = {
   id: string;
   projectId: string;
   name: string;
   message: string;
   stack: string;
-  level: IssueLevel;
+  level: IssueLevel | undefined;
   numEvents: number;
   numUsers: number;
+  status: IssueStatus | undefined;
```

Comment:
`level` and `status` should always be set so I wouldn't make them optional here. Fyi to make a field optional you can use the `?` operator:

```
level?: IssueLevel;
```